### DISCUSSION TITLE
Popover - Disable anchor moving when scrolling.

### DISCUSSION THREAD

**Discussion by Gambitboy** [tailwindlabs/headlessui#3233] [Help]
Hi, i'm using the popover component. My anchor is set to `bottom`. When I scroll to high up the anchor shifts to `top`

Any way to prevent it from moving? I'd like it just to stay bottom. Not bouncing between top and bottom based on scrolling.

**Comment by reoquitous** [comment]
Did you find a solution for this? I'm facing the same issue.

Edit: Found a hacky bandaid - Passing a negative gap to the anchor object prevents it from keeping itself forced inside the viewport.
```jsx
<PopoverPanel
  anchor={{
    to: 'bottom',
    gap: -200,
  }}
/>
```
Setting the value too high will make the panel entirely invisible though, so a proper solution would still be appreciated.
